packages/components/src/EditableText/EditableText.component.js

```diff
@@ -70,7 +85,7 @@ export function EditableTextComponent({ editMode, loading, inProgress, ...rest }
 			})}
 			{...allyProps}
 		>
-			<Component inProgress={inProgress} {...rest} />
+			<Component inProgress={inProgress} {...restWithoutFeature} />
```

Comment:
I think `restWithoutFeature` could contain `feature` if editMode is false? If that's the case I think it's better to rename to something else `componentProps` or anything?